### ISSUE TITLE
This is code is outdated in compare with PDF article "Learning to See: You Are What You See"

### ISSUE THREAD

**Issue by skabbit** [memo/webcam-pix2pix-tensorflow#9]
@memo , this is a great project! Thank you for making it real, and for sharing a code! I love original video "Learning to see" (as well as Diamanda Galas's voice!), and I'm trying to use same video instrument in my artwork with different datasets (themes). 

I've carefully read and found out, that this code has no "downscale" and "normalise" features.
But much more sad, that you have not released a code for pix2pix training! I mean that thing with random variation of parameters during image batch creation routine. Without that code whole this thing doesn't give results on quality level close to your masterpiece "Learning to see".

I'll appreciate if you release training code as well, to make this thing publicly available.
Thanks a lot!

**Comment by skabbit** [comment]
I've succeed only on facades dataset (making canny over the instance maps).
Also, I've adapted this code to python 3.6 and increased layers to generate 1024x1024 resolution.
